### PR TITLE
Add a caution for 2-node stretch clusters

### PR DESCRIPTION
**Pull request by danielharkerdp**
Using this article I have tried to deploy a 2-node Stretched File Server Cluster in Azure - not understanding that the storage must be detected by WSFC as shared storage, even though it is not actually shared.

### CODE REVIEW

**Comment by PRMerger14** [comment]
@danielharkerdp : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger15** [comment]
@danielharkerdp : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: WindowsServerDocs/storage/storage-replica/Stretch-Cluster-Replication-Using-Shared-Storage.md [hunk 1]
```diff
@@ -42,6 +42,9 @@ This walkthrough uses the following environment as an example:
 -   A network between servers with enough bandwidth to contain your IO write workload and an average of =5ms round trip latency, for synchronous replication. Asynchronous replication does not have a latency recommendation.
 -   The replicated storage cannot be located on the drive containing the Windows operating system folder.
 
+> [!IMPORTANT]
+> Whilst it is possible to attach a storage device to a single server and use this for replication, Windows Failover Clustering still relies upon SCSI Persistent Reservations. Therefore, the storage must still be a Shared Storage type such as a SAN technology. Local disks or disks presented by a hypervisor may not be compatible. In Azure, the disks must be a Premium SSD size that supports sharing, even if only one VM is to be attached to it.
```
Comment (illfated) [inline WindowsServerDocs/storage/storage-replica/Stretch-Cluster-Replication-Using-Shared-Storage.md:46]: Note that "whilst" may sound historical (maybe even archaic) to an English-speaking American person. "While" is sufficient in American English.
```suggestion
> While it is possible to attach a storage device to a single server and use this for replication, Windows Failover Clustering still relies upon SCSI Persistent Reservations. Therefore, the storage must still be a Shared Storage type such as a SAN technology. Local disks or disks presented by a hypervisor may not be compatible. In Azure, the disks must be a Premium SSD size that supports sharing, even if only one VM is to be attached to it.

```

**Comment by PRMerger19** [comment]
@danielharkerdp : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger20** [comment]
@danielharkerdp : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by illfated — APPROVED** [review]
Looks quite good to me.

**Comment by illfated** [comment]
#sign-off

**Comment by PRMerger16** [comment]
@illfated: I'm sorry - only the author of this article, @nedpyle, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by illfated** [comment]
Yeah, I know, I know. I just wanted to give this PR a little bit more attention. (The MS Docs team members might be gone on holiday already, so be prepared for some silent days ahead.

**Comment by PRMerger19** [comment]
@sethmanheim : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**